completions/just.bash

```diff
@@ -25,7 +25,15 @@ _just() {
                     COMPREPLY=( $(compgen -W "${opts}" -- "${cur}") )
                     return 0
                 elif [[ ${COMP_CWORD} -eq 1 ]]; then
-                    local recipes=$(just --summary --color never 2> /dev/null)
+                    if echo "${cur}" | grep -qF '/'; then
+                        local path_prefix=$(echo "${cur}" | sed 's/[/][^/]*$/\//')
+                        # TODO: Replace tr and awk below with --list-prefix "${path_prefix}". Need to update the --summary option to respect that first.
```

Comment:
naturally, I'd like to implement this. About that:

1. I don't know enough rust to implement it [well].
2. I don't know if there's a good reason for `--summary` to really ignore `--list-prefix`.

Reply:
Sorry for the delay in checking this out! `--summary` is intended for consumption by scripts. How would `--list-prefix` be used?

Reply:
COMPREPLY needs to have a full string to be completed. For example, when typing "ci/dep" then tab, COMPREPLY needs to say "ci/deploy" for completion to work. Right now, `just --summary ci/` would only say "deploy".

Using --list-prefix is one way to add "ci/" there. In retrospect, it's probably not the best way. Perhaps, --summary should include that automatically? This would be a breaking change for some scripts out there, unfortunately.

Reply:
See this answer for an alternative approach: https://stackoverflow.com/a/48663800

Reply:
ah, neat! That printf solution is less awk-ward than awk. :D

There's still the question of whether `just --summary` itself should include that prefix or not. What are your thoughts on that?

For now, I'll remove the TODO and change this to use printf.

Reply:
> There's still the question of whether just --summary itself should include that prefix or not. What are your thoughts on that?

Hmm, good question. So basically, the prefix is the directory prefix to get to the justfile, and thus invoke the recipe in question? I think it probably shouldn't, and it should be up to the caller who invoked just to add it.

Reply:
it would still be interesting to have an option to include the pull path, I suppose.